### DISCUSSION TITLE
Test certificates

### DISCUSSION THREAD

**Discussion by mexoboy** [go-acme/lego#1862] [Q&A] [answered]
Hi. Before I start using lego I used certbot client. Cert both have flag for receiving test certificates with increased rate limits. Is a lego have same functionality?

**Comment by ldez** [comment, marked as answer]
Hi,

yes, you can use the Let's Encrypt staging, you just have to set the staging server: `--server https://acme-staging-v02.api.letsencrypt.org/directory`

**Comment by mexoboy** [comment]
@ldez thanks a lot!
